RELEASING.md

```diff
@@ -0,0 +1,35 @@
+# Releasing
+To cut a new release of Anony follow these steps:
+
+1. Create a new branch for the release.
+```sh
+git checkout master
+git pull origin master
+git checkout -b v{VERSION}
+```
+
+2. Update the version of the library in `lib/anony/version.rb` following the
+   semantic versioning guidelines.
+
+3. Update the CHANGELOG.md file with the list of changes made since the last
+   release, including the version for the current changes. Please note that the
+   version must start with the character `v` in order to match the git tag that
+   will be pushed in a later step.
+
+4. Create a pull request for this release to merge the version branch into
+   master.
+
+5. Once the pull request is approved and merged, change branch to master and
```

Comment:
This is the manual way of tagging master, but you can also use GitHub releases - just creating a release will generate the correct tag for us, right... or do we need the explicit push so that CircleCI is triggered?

Reply:
CircleCI should work regardless of how the tag is created. I was not sure whether the GH release process created a tag. Will update the paragraph.

Reply:
Question: do we want to continue attaching binaries to GH releases?

Reply:
I've never bothered doing that myself, can't say that I've heard anybody asking for it, but at least with Rubygems being able to pull old versions is pretty easy to do so I've not needed it historically.